runtime/.eslintrc.js

```diff
@@ -1,10 +1,8 @@
 module.exports = {
-    extends: [
-        'plugin:@typescript-eslint/recommended',
-        'prettier/@typescript-eslint',
-    ],
+    extends: ['plugin:@typescript-eslint/recommended'],
     rules: {
         '@typescript-eslint/explicit-function-return-type': 'off',
         '@typescript-eslint/no-explicit-any': 'off',
+        'import/no-unresolved': 'off',
```

Comment:
Had to disable this rule for TS because it encourages imports without the file extension, whereas ES requires a file extension.

Reply:
Hmm.... we use bare filenames in other places, so this *should* work?  Can investigate later

Reply:
This is an aside we can discuss elsewhere but the gist of it is that we are going to see that bare filenames are going to fail static analysis when we update `cli-style` that has the import rules. Bare filenames are not part of the ES module spec, it's originally a Node thing that has crept into Webpack and other file loaders. The author of Node states that it is one of the deepest regrets he has about Node and I agree with him. It's a fundamentally bad idea that became popular because it's convenient to write (but not to read).

I hold the opinion that we should never do bare filenames, but it seems that TypeScript forces us to in at least some places, so disabling it for TS seems pragmatic in this case.

Reply:
OK, worth discussing separately as we use bare filenames extensively in javascript as well, but fine here!